src/middlewares/validators/createCommentsPayload.js

```diff
@@ -2,7 +2,7 @@ const Joi = require('joi');
 
 const createCommentsPayloadValidator = (req, res, next) => {
   const schema = Joi.object({
-    comment: Joi.string().min(2).max(30).required(),
+    comment: Joi.string().min(2).required(),
```

Comment:
Set min to be 1. There must not be comment length limits . It must be just not empty

Reply:
Fixed.